lib/tasks/user.rake

```diff
@@ -0,0 +1,38 @@
+namespace :hyku do
+  namespace :superadmin do
+    desc 'Create an initial superadmin user'
+    task create: :environment do
+      puts 'Creating an initial superadmin user.'
+      user = prompt_to_create_user
+      user.add_role(:superadmin)
+      puts 'User created.'
+    end
+
+    def prompt_to_create_user
```

Comment:
Since you're adding functionality, my preference is to go with this new rake task for both use cases.